randovania/games/fusion/logic_database/Sector 4 AQA.txt

```diff
@@ -97,12 +97,25 @@ Extra - room_id: [3]
               Diffusion Missile Data and Missiles
               All of the following:
                   # Freeze Middle Powamp
-                  Ice Missile Data and Missiles and Movement (Advanced)
-                  After Sector 4 Water Level Lowered or Damage Runs (Beginner)
+                  Knowledge (Advanced) and Movement (Advanced)
```

Comment:
this is now knowledge advancde + movement advanced, while the other is only knowledge advanced. they should be synced.